### ISSUE TITLE
Remove the "CHAT WITH YOUR SETUP SPECIALIST" message from Concierge

### ISSUE THREAD

**Issue by jamesdeanexpensify** [Expensify/App#54049]
## TL;DR
Coming from [this conversation](https://expensify.slack.com/archives/C07HPDRELLD/p1733943183942389), we'd like to remove this message entirely from Concierge:

<img width="797" alt="image" src="https://github.com/user-attachments/assets/ce5b3cb2-3db3-46dc-b291-2468248b1dd1" />

## Problem
The "CHAT WITH YOUR SETUP SPECIALIST" message from Concierge has confused multiple members to date because they then think the setup specialist themselves is in that chat (when it's only Concierge). And even though the message hyperlinks them to the #admins room, many don't click it. 

## Solution
Remove the "CHAT WITH YOUR SETUP SPECIALIST" message from Concierge entirely. It's unnecessary. We already have the `Meet your setup specialist task` from Concierge and also have the `Let's get you set up on Expensify` message from the setup specialist in the #admins room that will send as a custom subject line email.

**Comment by yuwenmemon** [comment]
This was deployed

**Comment by yuwenmemon** [comment]
Oh sorry it was just merged my bad.

**Comment by jamesdeanexpensify** [comment]
@yuwenmemon is this one good to close?

**Comment by yuwenmemon** [comment]
Yes
